src/app/ui/components/tooltip/basic-tooltip.tsx

```diff
@@ -15,7 +15,7 @@ export function BasicTooltip({ children, label, disabled, side }: BasicTooltipPr
   const isDisabled = !label || disabled;
   return (
     <Tooltip.Root>
-      <Tooltip.Trigger>{children}</Tooltip.Trigger>
+      <Tooltip.Trigger asChild>{children}</Tooltip.Trigger>
```

Comment:
Using `asChild` on every tooltip is going to cause accessibility issues, see: https://www.radix-ui.com/primitives/docs/guides/composition#changing-the-element-type. We need to set asChild only when BasicTooltip wraps an interactive tag like `a` or `button`.

I've recently opened a PR that addresses tooltip issues here: #4858